locale/it/index.md

```diff
@@ -6,10 +6,10 @@ labels:
   download-for: Download per
   other-downloads: Altri Downloads
   other-lts-downloads: Altri Download LTS
-  other-stable-downloads: Altri Download Stabili
-  stable: Stabile
+  other-current-downloads: Altri Download Stabili
```

Comment:
"Altri download della versione corrente" or "Altri download correnti".
Not sure about casing.

cc: @parro-it @piccoloaiutante

P.S. we can do this things in another pr :)